content/docs/user-guide/engine/_index.md

```diff
@@ -1,14 +1,34 @@
 ---
-description: The following topics will guide you through programming with and for O3DE. 
+description: Learn about Open 3D Engine's core systems and how you can extend the systems by programming.
 title: Open 3D Engine Core Programming
 linktitle: Engine Core
 weight: 600
 ---
 
-## Topics
+The *engine core* encompasses the source code, systems, and modules that underly **Open 3D Engine (O3DE)** and its tools. All of these tools -- Editor, Asset Processor, Script Canvas, and so on -- work together to build an interactive 3D experience that uses modern rendering techniques and game play logic, and move your design from an empty scene to a fully polished project that you can deploy and share. In this section, you'll gain a deeper understanding in each part of the engine core and learn how you can expand O3DE's toolkit.
```

Comment:
```suggestion
The *engine core* encompasses the source code, systems, and modules that underlie **Open 3D Engine (O3DE)** and its tools. All of these tools---**O3DE Editor**, **Asset Processor**, **Script Canvas**, and so on---work together to build an interactive 3D experience that uses modern rendering techniques and gameplay logic. Using these tools, you can take your design from an empty scene to a fully polished project that you can deploy and share. In this section, you'll gain a deeper understanding of each part of the engine core and learn how you can expand O3DE's toolkit.
```
* Spelling: "underlie"
* Punctuation: The two dashes render as an en dash. To set aside a text like this, use em dashes with no space on either side. (I'm not sure if three hyphens renders as an em dash, but that's the suggestion here.)
* Tool names: Per our [terminology guidelines](https://docs.o3de.org/docs/contributing/to-docs/terminology/):
  * "Use **Bold** text for the first on-page reference of a tool...."
  * Use "**O3DE Editor**" in bold for the first on-page reference.
  * [For Asset Processor]: "Use bold text for the first on-page reference."
  * Apply the same rules to "Script Canvas".
* The second sentence is *very* long. Suggested edit to split it into two sentences.
* Changed "understanding in" to "understanding of"